nav2_system_tests/src/costmap_filters/CMakeLists.txt

```diff
@@ -5,10 +5,9 @@ ament_add_test(test_keepout_filter
   TIMEOUT 180
   ENV
     TEST_DIR=${CMAKE_CURRENT_SOURCE_DIR}
-    TEST_MAP=${PROJECT_SOURCE_DIR}/maps/map_circular.yaml
-    TEST_MASK=${PROJECT_SOURCE_DIR}/maps/keepout_mask.yaml
-    TEST_WORLD=${PROJECT_SOURCE_DIR}/worlds/turtlebot3_ros2_demo.world
```

Comment:
and world?

Reply:
I replaced the variables TEST_MAP and TEST_WORLD with get_package_share_directory ('aws_robomaker_small_warehouse_world') because I assume aws_robomaker_small_warehouse_world package is installed from binaries.